src/js/main.js

```diff
@@ -429,41 +414,37 @@ function startProcess() {
 
     $("#content").on('keydown', 'input[type="number"]', function (e) {
         // whitelist all that we need for numeric control
-        var whitelist = [
+        const whitelist = [
             96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, // numpad and standard number keypad
             109, 189, // minus on numpad and in standard keyboard
             8, 46, 9, // backspace, delete, tab
             190, 110, // decimal point
-            37, 38, 39, 40, 13 // arrows and enter
+            37, 38, 39, 40, 13, // arrows and enter
         ];
 
-        if (whitelist.indexOf(e.keyCode) == -1) {
+        if (whitelist.indexOf(e.keyCode) === -1) {
             e.preventDefault();
         }
     });
 
     $("#content").on('change', 'input[type="number"]', function () {
-        var element = $(this),
-            min = parseFloat(element.prop('min')),
-            max = parseFloat(element.prop('max')),
-            step = parseFloat(element.prop('step')),
-            val = parseFloat(element.val()),
-            decimal_places;
+        const element = $(this),
+              min = parseFloat(element.prop('min')),
+              max = parseFloat(element.prop('max')),
+              step = parseFloat(element.prop('step'));
+
```

Comment:
Similar complaint as above. Actually I'll look into it.

Reply:
This simple states that the variable can't be changed later. Like final in other languages like Java.

Reply:
I was thinking that maybe like this would be more readable. 
```javascript
        const element = $(this);
        const min = parseFloat(element.prop('min'));
        const max = parseFloat(element.prop('max'));
        const step = parseFloat(element.prop('step'));
```

Reply:
To me is not a problem to define the type only one, with all the names after that. I like to do it in this way. But I'm open to any decision. Let's see what others think about it.